<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="HomeBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>home</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>438</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="RegistrationBox">
    <item>
     <widget class="QLabel" name="RegistrationLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Registration                                                                              </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="UsernameForm">
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EmailForm">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EmailReEntryForm">
      <property name="placeholderText">
       <string>Email (again)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PasswordForm">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PasswordReEntryForm">
      <property name="placeholderText">
       <string>Password (again)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RegisterBtn">
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>266</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="HaveAnAccountLabel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Have an account?                                                                                         </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SignInBtn">
      <property name="text">
       <string>Sign In</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
